app/repositories/SessionCacheRepository.scala

```diff
@@ -104,6 +105,10 @@ trait SessionCacheRepository {
 case class AccountLinksMongo(accountLinks: Seq[AccountLink], lastUpdated: LocalDateTime = LocalDateTime.now)
 
 object AccountLinksMongo {
+
+  def unapply(alm: AccountLinksMongo): Option[(Seq[AccountLink], LocalDateTime)] =
```

Comment:
Is there any specific reason to create this method as at line no 116 (unlift(AccountLinksMongo.unapply)) can be replaced with below ?

(accLinks => Tuple.fromProductTyped(accLinks))